src/main/java/Serghei_Stoianov/BreakAndContinue.java

```diff
@@ -0,0 +1,26 @@
+package Serghei_Stoianov;
+
+import java.util.Scanner;
+
+public class BreakAndContinue {
+    public static void main(String[] args) {
+        int i;
+
+////Create a program, which will ask user to enter a number, if number will be negative exit the loop
+        Scanner scanner = new Scanner(System.in);
+        for (i = 1; i++ > 0; ) {
+            System.out.println("enter a number");
+            int sc = scanner.nextInt();
+            if (sc < 0) break;
+        }
+
+
+//Create a program which will print all number from 1 to 10 except 3 and 7 using continue keyword
+        for (i = 0; i++ < 10; ) {
```

Comment:
please move i++ to its place in for loop

Reply:
in this solution it does not affect the result, but i've change it place.
done

Reply:
it will not affect but it is weird because it is not while loop where you don't have a special block where you can specify incrementation it is for where you have this